### ISSUE TITLE
tvOS 10.1

### ISSUE THREAD

**Issue by bastiaandevogel** [jeffkreeftmeijer/UitzendingGemist#34]
Getting some errors. I did not "convert" as xcode suggested.
Can this be fixed? Or did I not configure it right?

2017-03-04 23:43:38.654804 Uitzending Gemist[168:4089] [User Defaults] Failed to write value for key NetworkTimingMetrics in CFPrefsPlistSource<0x17411fda0> (Domain: com.apple.itunesstored, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null)): setting preferences outside an application's container requires user-preference-write or file-write-data sandbox access, switching to read-only
2017-03-04 23:43:38.662349 Uitzending Gemist[168:4089] [User Defaults] attempt to set (null) for key in NetworkTimingMetrics in read-only (due to a previous failed write) preferences domain CFPrefsPlistSource<0x17411fda0> (Domain: com.apple.itunesstored, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null))
2017-03-04 23:43:38.817024 Uitzending Gemist[168:4089] [User Defaults] attempt to set (null) for key in NetworkTimingMetrics in read-only (due to a previous failed write) preferences domain CFPrefsPlistSource<0x17411fda0> (Domain: com.apple.itunesstored, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null))
2017-03-04 23:43:39.076790 Uitzending Gemist[168:4152] SSURLSessionManager: A nil value was loaded from the bag for key metrics.
2017-03-04 23:43:39.083518 Uitzending Gemist[168:4089] SSURLSessionManager: A nil value was loaded from the bag for key metrics.
2017-03-04 23:43:39.084152 Uitzending Gemist[168:4153] [User Defaults] attempt to set (null) for key in NetworkTimingMetrics in read-only (due to a previous failed write) preferences domain CFPrefsPlistSource<0x17411fda0> (Domain: com.apple.itunesstored, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null))
2017-03-04 23:43:39.278480 Uitzending Gemist[168:4153] SSURLSessionManager: A nil value was loaded from the bag for key metrics.
2017-03-04 23:43:39.278782 Uitzending Gemist[168:4152] SSURLSessionManager: A nil value was loaded from the bag for key metrics.
2017-03-04 23:43:39.279168 Uitzending Gemist[168:4089] SSURLSessionManager: A nil value was loaded from the bag for key metrics.
2017-03-04 23:43:39.289642 Uitzending Gemist[168:4154] [User Defaults] attempt to set (null) for key in NetworkTimingMetrics in read-only (due to a previous failed write) preferences domain CFPrefsPlistSource<0x17411fda0> (Domain: com.apple.itunesstored, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null))
2017-03-04 23:43:39.297931 Uitzending Gemist[168:4152] [User Defaults] attempt to set (null) for key in NetworkTimingMetrics in read-only (due to a previous failed write) preferences domain CFPrefsPlistSource<0x17411fda0> (Domain: com.apple.itunesstored, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null))
2017-03-04 23:43:39.357132 Uitzending Gemist[168:4153] SSURLSessionManager: A nil value was loaded from the bag for key metrics.
2017-03-04 23:43:39.357157 Uitzending Gemist[168:4151] SSURLSessionManager: A nil value was loaded from the bag for key metrics.

**Comment by hollanderbart** [comment]
"<!DOCTYPE HTML PUBLIC \"-//IETF//DTD HTML 2.0//EN\">\n<html><head>\n<title>410 Gone</title>\n</head><body>\n<h1>Gone</h1>\n<p>The requested resource<br />/npoplayer/i.js<br />\nis no longer available on this server and there is no forwarding address.\nPlease remove all references to this resource.</p>\n</body></html>\n"

**Comment by HarmElz** [comment]
The following can be used to fix the issue with the new API:

update the following files (as attached) on the server side. This also includes LIVE NPO 1, 2 and 3.

[uzg.zip](https://github.com/jeffkreeftmeijer/UitzendingGemist/files/824616/uzg.zip)

**Comment by hollanderbart** [comment]
I've created a cocoapod library which I already updated to the new NPO API.
https://github.com/hollanderbart/NPOStream

**Comment by bforbird** [comment]
@HarmElz Yes that did it! It works! I didn't get the livestream working yet, but that's ok.

**Comment by jeffkreeftmeijer** [comment]
@HarmElz thanks for your work on this! Could you create a pull request with your changes to support the new API? 

Also, although I'm using https://github.com/Mauricevb/NPO-Live-Apple-TV-4I'm open to look into merging the livestreams. Could you separate that from the new API code and submit that as a separate pull request?

Thanks!

**Comment by HarmElz** [comment]
Hi Jeff,

I made pull request, but made a small mistake when committing the last change for enabling the Live part. This is the addition of the Live menu option in the MenuBar.xml.js file.

I used the Mauricevb/NPO Live app before, but this app stopped working for me as well, so that was the reason to include it into this app as well, because that was easier for me than fixing 2 app's.
